deployment/CP05MOAS-GL389_Deploy.csv

```diff
@@ -29,9 +29,9 @@ P10b,,AR-31,,CP05MOAS-GL389-02-FLORTM000,5,1,2018-07-24T16:20:30,2018-10-26T00:3
 P10b,,AR-31,,CP05MOAS-GL389-03-CTDGVM000,5,1,2018-07-24T16:20:30,2018-10-26T00:37:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-CTDGVM-09349,39.88338,-70.64638,,1000,1000,FZ-2
 P10b,,AR-31,,CP05MOAS-GL389-04-DOSTAM000,5,1,2018-07-24T16:20:30,2018-10-26T00:37:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-DOSTAM-00189,39.88338,-70.64638,,1000,1000,FZ-2
 P10b,,AR-31,,CP05MOAS-GL389-05-PARADM000,5,1,2018-07-24T16:20:30,2018-10-26T00:37:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-PARADM-50222,39.88338,-70.64638,,1000,1000,FZ-2
-TS20190205,,,,CP05MOAS-GL389-00-ENG000000,6,1,2019-02-05T08:17:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGCON-GLDRCP-00389,39.9161,-70.369722,,1000,1000,SS-2
-TS20190205,,,,CP05MOAS-GL389-01-ADCPAM000,6,1,2019-02-05T08:17:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-ADCPAM-00539,39.9161,-70.369722,,1000,1000,SS-2
-TS20190205,,,,CP05MOAS-GL389-02-FLORTM000,6,1,2019-02-05T08:17:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-FLORTM-04198,39.9161,-70.369722,,1000,1000,SS-2
-TS20190205,,,,CP05MOAS-GL389-03-CTDGVM000,6,1,2019-02-05T08:17:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-CTDGVM-09349,39.9161,-70.369722,,1000,1000,SS-2
-TS20190205,,,,CP05MOAS-GL389-04-DOSTAM000,6,1,2019-02-05T08:17:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-DOSTAM-00189,39.9161,-70.369722,,1000,1000,SS-2
-TS20190205,,,,CP05MOAS-GL389-05-PARADM000,6,1,2019-02-05T08:17:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-PARADM-50222,39.9161,-70.369722,,1000,1000,SS-2
\ No newline at end of file
+TS20190205,,,,CP05MOAS-GL389-00-ENG000000,6,1,2019-02-05T08:17:00,2019-05-10T09:51:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGCON-GLDRCP-00389,39.9161,-70.369722,,1000,1000,SS-2
+TS20190205,,,,CP05MOAS-GL389-01-ADCPAM000,6,1,2019-02-05T08:17:00,2019-05-10T09:51:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-ADCPAM-00539,39.9161,-70.369722,,1000,1000,SS-2
+TS20190205,,,,CP05MOAS-GL389-02-FLORTM000,6,1,2019-02-05T08:17:00,2019-05-10T09:51:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-FLORTM-04198,39.9161,-70.369722,,1000,1000,SS-2
+TS20190205,,,,CP05MOAS-GL389-03-CTDGVM000,6,1,2019-02-05T08:17:00,2019-05-10T09:51:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-CTDGVM-09349,39.9161,-70.369722,,1000,1000,SS-2
+TS20190205,,,,CP05MOAS-GL389-04-DOSTAM000,6,1,2019-02-05T08:17:00,2019-05-10T09:51:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-DOSTAM-00189,39.9161,-70.369722,,1000,1000,SS-2
+TS20190205,,,,CP05MOAS-GL389-05-PARADM000,6,1,2019-02-05T08:17:00,2019-05-10T09:51:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-PARADM-50222,39.9161,-70.369722,,1000,1000,SS-2
```

Comment:
same as 336 - maybe a note explaining that glider was actually lost.

Reply:
I concur with Rebecca.  Please add a note to line 32 noting that the glider was lost.  Perhaps put "N/A" in the recovery cruise column so that we know that space is intentionally left blank.